docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: "3.8"
+services:
+  tests:
+    image: buildkite/plugin-tester
```

Just realized that you should pin the version here just in case:
```suggestion
    image: buildkite/plugin-tester:v3.0.1
```